### PR TITLE
fixes 2 bugs:

### PR DESCRIPTION
**Pull request by Isentrope1**
- ganache 1.9.1 is broken. need to use ^1.9.2
- sent ETH (100) was too much. lowered to 40

### CODE REVIEW

**Review by mikhaelsantos — APPROVED** [review]
LGTM

**Comment by fonty1** [comment]
> * ganache 1.9.1 is broken. need to use ^1.9.2

Do you mean ^2.9.2?

**Review by fonty1 — APPROVED** [review]
👍

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "license": "ISC",
   "dependencies": {
     "ethers": "^4.0.33",
-    "ganache-core": "^2.6.1",
```
Comment (mikhaelsantos) [inline package.json:13]: Wouldn't just  work to put npm ci, instead of npm install!

**Review by mikhaelsantos — CHANGES_REQUESTED** [review]

**Comment by mikhaelsantos** [comment]
A pull request was merged that I think makes this PR unecesary. @isentropy please double check.

**Review by jtakalai — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (jtakalai) [reply package.json:13]: I removed the hats in another branch

Comment (mikhaelsantos) [reply package.json:13]: I fixed this already with my PR

**Comment by mikhaelsantos** [comment]
This PR seems to fix all the issues and I added you 40 ETH change:
https://github.com/streamr-dev/streamr-ganache/pull/9

@isentropy test what is deployed and see if this PR can be closed

**Comment by Isentrope1** [comment]
Can I close this @mikhaelsantos ? Looks like you already merged fix

**Comment by mikhaelsantos** [comment]
Yes!

**Pull request closed without merging**